api/server.spec.js

```diff
@@ -0,0 +1,82 @@
+const request = require('supertest');
+const server = require('./server.js');
+const db = require('../data/dbConfig.js');
+
+describe('the server', () => {
+  
+  describe('the environment', () => {
+    it('should set testing environment', () => {
+      const env = process.env.DB_ENV;
+
+      expect(env).toBe('testing');
+    });
+  });
+ 
+  describe('GET / Testing', () => {
+    it('should return status 200', async () => {
+      const res = await request(server).get('/');
```

Comment:
The three `it()` functions in this describe can be combined into one. You're testing different pieces of the same request so it would be better to just have one `it()` function that checks each part of the response.